.github/ISSUE_TEMPLATE/j_design_pattern.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Add design pattern
+about: Epic ticket for adding a new design pattern
+title: "[Design pattern] %label of CL class%"
+assignees: 
+labels:
```

Comment:
Add the label `pattern`? If you search this label, there's a ticket (from 2016) related to this issue template.